app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
My thought right now would be `/audit-events/services/<service_id>/acknowledge?event-id=<event_id>`?

(I'm finding the 'including-previous' thing a bit hard to parse, and maybe it's easier to just document that if you acknowledge an event, you're also acknowledging all earlier events on the same service?)